components/analytics/system_analytics/system_analytics.tsx

```diff
@@ -31,13 +32,19 @@ const StatTypes = Constants.StatTypes;
 type Props = {
     isLicensed: boolean;
     stats?: Record<string, number | AnalyticsRow[]>;
+    pluginStatHandlers: GlobalState['plugins']['siteStatsHandlers'];
+}
+
+type State = {
+    pluginSiteStats: Record<string, PluginAnalyticsRow>;
 }
 
-export default class SystemAnalytics extends React.PureComponent<Props> {
-    private pluginStats: Record<string, PluginAnalyticsRow>;
+export default class SystemAnalytics extends React.PureComponent<Props, State> {
     public constructor(props: Props) {
         super(props);
-        this.pluginStats = {};
+        this.state = {
+            pluginSiteStats: {},
+        };
```

Comment:
We can get rid of the constructor and just write

```js
state = {
    pluginStats: {},
}
```

Reply:
changed! didn't know that this is possible :)